### PR TITLE
0.9.0 Getting started for deb distribution

### PR DESCRIPTION
**Pull request by drigodwin**
See [PR40](https://github.com/apache/brooklyn-docs/pull/40)

### CODE REVIEW

File: guide/start/running.md [hunk 1]
```diff
@@ -68,7 +69,24 @@ $ yum install apache-brooklyn-{{site.brooklyn-version}}-1.rpm
...
-<strong class="hidden started-pdf-include">c) OSX / Linux</strong>
+<strong class="hidden started-pdf-include">c) Debian / Ubuntu</strong>
+
+For Ubuntu and Debian users, the recommended way to install Apache Brooklyn is to use the deb file. 
+
+The deb file is the de facto standard for packaging software on these Linux distributions and provides a mechanism for installing, upgrading and removing packages such as Apache Brooklyn. The deb package contains all the necessary files associated with the Apache Brooklyn application. 
+
+Download the Apache Brooklyn [deb distribution](https://www.apache.org/dyn/closer.lua/brooklyn/apache-brooklyn_{{site.brooklyn-version}}_noarch.deb){:target="_blank"}.
+
+Once downloaded, run the following shell command:
```
Comment (neykov) [inline guide/start/running.md]: I'd include something in the lines of - "make sure you have the `default-jre-headless` package installed" or include it in the snippet below as an apt-get cmd.

File: guide/start/running.md [hunk 2]
```diff
@@ -154,7 +173,20 @@ The application should then output its logs to `/var/log/brooklyn/apache-brookly
...
-<strong class="hidden started-pdf-include">c) OSX / Linux</strong>
+<strong class="hidden started-pdf-include">c) Ubuntu / Debian</strong>
+
+Apache Brooklyn should now have been installed and be running as a system service. It can stopped and started with the standard service commands:
+
+{% highlight bash %}
+$ sudo service brooklyn start|stop|restart|status
+{% endhighlight %}
+
+The application should then output its logs to `/var/log/brooklyn/apache-brooklyn.debug.log` and `/var/log/brooklyn/apache-brooklyn.info.log`.
```
Comment (neykov) [inline guide/start/running.md]: Log file name is called `brooklyn.{debug, info}.log` atm.

File: guide/start/running.md (hunk 1 above)
Comment (drigodwin) [inline guide/start/running.md]: The top of the page says "The latter assumes that you have a Java Runtime Environment (JRE) installed (version 7 or later)" as this is the case for all of the non vagrant install options. I don't think it's worth putting this in seperately for all of the install options.

Comment (neykov) [inline guide/start/running.md]: Having the java package installed (say openjdk-7-jre) is not enough unfortunately, dpkg will fail without adding `default-jre-headless`.

**Comment by drigodwin** [comment]
Not valid for the 0.9.0 branch

**Pull request closed without merging**